tests.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import random
+import unittest
+from collections import Counter
+
+from weightedDict import WeightedDict
+
+
+class TestWeightedDict(unittest.TestCase):
+    # Usage example
```

Comment:
Hi James, Thanks for contributing!  Most of the Google code uses 2 spaces for indents.   This is my preference too, but I'm not strongly attached one way or the other.